examples/vision/xray_classification_with_tpus.py

```diff
@@ -37,39 +37,44 @@
 """
 
 AUTOTUNE = tf.data.experimental.AUTOTUNE
-GCS_PATH = "gs://kds-7c9306925365b635aa934a70a0d94688c717d8c2eda0e47466736307"
 BATCH_SIZE = 25 * strategy.num_replicas_in_sync
 IMAGE_SIZE = [180, 180]
+CLASS_NAMES = ["NORMAL", "PNEUMONIA"]
 
 """
 ## Load the data
 
 The Chest X-ray data we are using from
-[*Cell*](https://www.kaggle.com/paultimothymooney/chest-xray-pneumonia) divides the data
-into training, validation, and test files. There are only 16 files in the validation folder,
-and we would prefer to have a less extreme division between the training and the validation set.
-We will append the validation files and create a new split that resembles the standard
-80:20 division instead.
-"""
+[*Cell*](https://www.cell.com/cell/fulltext/S0092-8674(18)30154-5) divides the data into
+training and test files. Let's first load in the training TFRecords.
 
-filenames = tf.io.gfile.glob(str(GCS_PATH + "/chest_xray/train/*/*"))
-filenames.extend(tf.io.gfile.glob(str(GCS_PATH + "/chest_xray/val/*/*")))
+Note: The TFRecord contains a DS_Store file that is not part of the X-ray dataset. Let's
```

Comment:
This is unfortunate. Prefer reuploading the data without this file. It is not related to the dataset.

Reply:
Okay I will ask Mark if he can delete it for us.

Reply:
The dataset has been updated

Reply:
Great! Please remove the comment above.